src/main/kotlin/com/exactpro/th2/codec/AbstractCodecProcessor.kt

```diff
@@ -43,7 +43,9 @@ abstract class AbstractCodecProcessor(
     }
 
     protected fun String?.onErrorEvent(message: String, messagesIds: List<MessageID> = emptyList(), cause: Throwable? = null) {
-        logger.error(cause) { "$message. Messages: ${messagesIds.joinToString(", ", transform = MessageID::toDebugString)}" }
+        logger.error(cause) { "$message. Messages: ${messagesIds.joinToString(", ") {
+            "${it.connectionId.sessionAlias}.${it.direction}.${it.sequence}.[${it.subsequenceList.joinToString(",")}]"
```

Comment:
Maybe use `:` instead of `.` as a separator (except for subsequences)? I.e.: `session-alias:direction:sequence[.subsequence]*`
This will allow us to use these ids in rpt-data-provider without any modifications